### PR TITLE
[OSCI] Add New Author shashwat

### PR DESCRIPTION
**Pull request by AwesomeSauce42**
### Description
Added New author Shashwat 

### Issues Resolved
N/A
### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/shashwat.markdown [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Shashwat Prakash
+short_name: Shash
+twitter: Awesome01768974
+github: AwesomeSauce42
+---
+
+**Shashwat** loves to play Badminton and chess. Hobbies including playing the guitar and piano. Major interests would inclued watching shows!
```
Comment (joshuarrrr) [inline _authors/shashwat.markdown:8]: A couple minor style fixes:
```suggestion
**Shashwat** loves to play badminton and chess. Hobbies including playing the guitar and piano. Major interests would include watching shows!
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
@AwesomeSauce42 Please move the new file to the `_authors/osci_2023` directory.

**Comment by dtaivpp** [comment]
@AwesomeSauce42 please address Josh's comments and we should be good to go!

**Comment by AwesomeSauce42** [comment]
> @AwesomeSauce42 please address Josh's comments and we should be good to go!

Hi sir, I've made the changes mentioned by sir Josh. Please review!

File: _authors/osci_2023/shashwat.markdown [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Shashwat Prakash
+short_name: Shash
+twitter: Awesome01768974
+github: AwesomeSauce42
+---
+
+**Shashwat** loves to play adminton and chess. Hobbies include playing the guitar and piano. My primary interests include watching shows, gaming, and coding!
```
Comment (joshuarrrr) [inline _authors/osci_2023/shashwat.markdown:8]: nit
```suggestion
**Shashwat** loves to play badminton and chess. Hobbies include playing the guitar and piano. My primary interests include watching shows, gaming, and coding!
```

**Review by AwesomeSauce42 — COMMENTED** [review]
Typo fixed.

**Comment by AwesomeSauce42** [comment]
@joshuarrrr could you confirm the changes? Thanks!

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by dtaivpp — APPROVED** [review]
Looks good to me!

**Pull request merged**